heat/core/trigonometrics.py

```diff
@@ -99,6 +102,44 @@ def arctan(x, out=None):
     return local_op(torch.atan, x, out)
 
 
+def arctan2(x1, x2, out=None):
```

Comment:
Charlie updated our docstrings (to be found in branch 504). This function needs

a) a return type annotation
b) ``DNDarray`` should be :class:`DNDarray`

Also out should be either removed if not used/applicable